prometheus.lua

```diff
@@ -447,7 +447,7 @@ local function lookup_or_create(self, label_values)
 
   self.lookup_size = self.lookup_size + 1
 
-  local err = self._key_index:add(full_name, ERR_MSG_LRU_EVICTION)
+  local err = self._key_index:add(full_name, ERR_MSG_LRU_EVICTION, self.exptime)
```

Comment:
I think the name `ttl` is shorter and easier to understand

Reply:
I just followed the openresty naming style
![Uploading image.png…]()